app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/LoginScreen.kt

```diff
@@ -56,10 +58,21 @@ fun LoginScreen(
     LoginAppLogo(modifier = Modifier.weight(1f))
     WelcomeTitle(modifier = Modifier.weight(4f))
     SignInButton(modifier = Modifier.weight(1f)) {
-      val intent =
-          AuthorizationClient.createLoginActivityIntent(
-              context.getActivity(), viewModel.getAuthorizationRequest())
-      launcher.launch(intent)
+      // val intent =
+      //     AuthorizationClient.createLoginActivityIntent(
+      //         context.getActivity(), viewModel.getAuthorizationRequest())
+      // launcher.launch(intent)
```

Comment:
commented code

Reply:
I mean this is just for testing, I will delete it for the final pr